.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "arrowParens": "always",
+  "htmlWhitespaceSensitivity": "ignore"
```

Comment:
I did modify both these Prettier rules:
- https://prettier.io/docs/en/options.html#arrow-function-parentheses
   - (always place optional perens `(like) => "this"`
- https://prettier.io/docs/en/options.html#html-whitespace-sensitivity
   - I believe this stops it from adding extra `{" "}` whenever we reformat `display:inline` elements